### PR TITLE
fix: add missing container images in VHD

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**Reason for Change**:

Noticed that Tiller 2.11.0 and blobfuse-flexvolume addon container images were missing from the VHD install script. This PR adds them.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
lgtm

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -87,7 +87,7 @@ for PAUSE_VERSION in ${PAUSE_VERSIONS}; do
     pullContainerImage "docker" "k8s.gcr.io/pause-amd64:${PAUSE_VERSION}"
 done
 
-TILLER_VERSIONS="2.8.1"
+TILLER_VERSIONS="2.8.1 2.11.0"
```
Comment (jackfrancis) [inline packer/install-dependencies.sh]: bummer

File: packer/install-dependencies.sh [hunk 2]
```diff
@@ -142,9 +142,9 @@ for KV_FLEXVOLUME_VERSION in ${KV_FLEXVOLUME_VERSIONS}; do
     pullContainerImage "docker" "mcr.microsoft.com/k8s/flexvolume/keyvault-flexvolume:v${KV_FLEXVOLUME_VERSION}"
 done
 
-BLOBFUSE-FLEXVOLUME_VERSIONS="1.0.7"
-for FLEXVOLUME_VERSIONS in ${FLEXVOLUME_VERSIONS}; do
-    pullContainerImage "docker" "mcr.microsoft.com/k8s/flexvolume/blobfuse-flexvolume:v${FLEXVOLUME_VERSIONS}"
+BLOBFUSE_FLEXVOLUME_VERSIONS="1.0.7"
+for BLOBFUSE_FLEXVOLUME_VERSION in ${BLOBFUSE_FLEXVOLUME_VERSIONS}; do
```
Comment (mboersma) [inline packer/install-dependencies.sh]: Wow, nice catch.

Comment (CecileRobertMichon) [reply packer/install-dependencies.sh]: actually, CI caught it ;)

Comment (mboersma) [reply packer/install-dependencies.sh]: I sure didn't.

**Pull request merged**